doc/ref_arch/kubernetes/chapters/chapter03.rst

```diff
@@ -259,15 +259,16 @@ vendor-specific activation and life cycle management, and securely maps these de
 
 Figure 3-2 shows in four steps how device plugins operate on a Kubernetes node:
 
--  1: During setup, the cluster administrator (more in :ref:`ref_arch/kubernetes/chapters/chapter03:3.2.5.1 Operator Pattern `)
-   knows or discovers (as per :ref:`ref_arch/kubernetes/chapters/chapter03:3.2.1.5 Node Feature Discovery `) what kind of
-   devices are present on the different nodes, selects which devices to enable and deploys the associated device
-   plugins.
--  2: The plugin reports the devices it found on the node to the Kubelet device manager and starts its gRPC server
-   to monitor the devices.
--  3: A user submits a pod specification (workload manifest file) requesting a certain type of device.
--  4: The scheduler determines a suitable node based on device availability and the local kubelet assigns a specific
-   device to the pod's containers.
+- 1: During setup, the cluster administrator
+  (more in :ref:`ref_arch/kubernetes/chapters/chapter03:3.2.5.1 Operator Pattern`)
```

Comment:
Since this is within the Chapter, the reference can be simplified to"  `Operator Pattern`_ " but even without that the numbering should be deleted.

The same is true for other references.